daemon/oci_linux_test.go

```diff
@@ -203,6 +204,8 @@ func TestSysctlOverrideHost(t *testing.T) {
 }
 
 func TestGetSourceMount(t *testing.T) {
+	testutil.SkipWhenUnprivileged(t)
```

Comment:
How is this failing?
I'm guessing under certain conditions when evaluating symlinks?